data/json/monsters/insect_spider.json

```
@@ -842,7 +842,7 @@
     "anger_triggers": [ "PLAYER_CLOSE" ],
     "fear_triggers": [ "HURT" ],
     "flags": [ "AQUATIC", "SEES", "WATER_CAMOUFLAGE" ],
-    "upgrades": { "age_grow": 21, "into": "mon_dragonfly_giant" }
+    "upgrades": { "half_life": 42, "into": "mon_dragonfly_giant" }
```

```suggestion
    "upgrades": { "age_grow": 42, "into": "mon_dragonfly_giant" }
```
The general problem of evo + reproduction hits dragonflies as well - naiads would stop existing, and fused dragonflies would essentially clone themselves as long as they drop the eggs in water.